SampleTests: autoCompleteSampleTest failed

Note: #5060 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: ac54f30d5e5c751a16c39c5b771ddac596e60cc6
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/7c40da8c-4445-49fa-8acf-a60d7f92bda9), [Sponge](http://sponge2/7c40da8c-4445-49fa-8acf-a60d7f92bda9)
status: failed
<details><summary>Test output</summary><br><pre>expected to contain a match for:
    .*completionResults.*"suggestion":"Google","type":"COMPANY_NAME"}.*
    .*completionResults.*"suggestion":"Software Engineer","type":"JOB_TITLE".*
    .*completionResults.*"suggestion":"Software Engineer","type":"JOB_TITLE".*
    
but was:
    Company generated: GenericData{classInfo=[careerSiteUri, derivedInfo, displayName, eeoText, externalId, headquartersAddress, hiringAgency, imageUri, keywordSearchableJobCustomAttributes, name, size, suspended, websiteUri], {displayName=Google, externalId=company:8548476233227929101, headquartersAddress=1600 Amphitheatre Parkway Mountain View, CA 94043}}
    Company created: {"derivedInfo":{"headquartersLocation":{"latLng":{"latitude":37.4219891,"longitude":-122.0840944},"locationType":"STREET_ADDRESS","postalAddress":{"addressLines":["1600 Amphitheatre Pkwy, Mountain View, CA 94043, USA"],"administrativeArea":"CA","locality":"Mountain View","postalCode":"94043","regionCode":"US"},"radiusInMiles":7.391165065886471E-5}},"displayName":"Google","externalId":"company:8548476233227929101","headquartersAddress":"1600 Amphitheatre Parkway Mountain View, CA 94043","name":"projects/java-docs-samples-testing/companies/e008c43f-33c3-4b45-869f-18b914baa87f"}
    Job generated: GenericData{classInfo=[addresses, applicationInfo, companyDisplayName, companyName, compensationInfo, customAttributes, degreeTypes, department, derivedInfo, description, employmentTypes, incentives, jobBenefits, jobEndTime, jobLevel, jobStartTime, languageCode, name, postingCreateTime, postingExpireTime, postingPublishTime, postingRegion, postingUpdateTime, processingOptions, promotionValue, qualifications, requisitionId, responsibilities, title, visibility], {applicationInfo=GenericData{classInfo=[emails, instruction, uris], {uris=[http://careers.google.com]}}, companyName=projects/java-docs-samples-testing/companies/e008c43f-33c3-4b45-869f-18b914baa87f, description=Design, develop, test, deploy, maintain and improve software., requisitionId=jobWithRequiredFields:474884892813030639, title=Software Engineer}}
    Job created: {"applicationInfo":{"uris":["http://careers.google.com"]},"companyDisplayName":"Google","companyName":"projects/java-docs-samples-testing/companies/e008c43f-33c3-4b45-869f-18b914baa87f","description":"Design, develop, test, deploy, maintain and improve software.","languageCode":"en","name":"projects/java-docs-samples-testing/jobs/97817947029807814","postingCreateTime":"2021-08-21T19:20:39.397Z","postingExpireTime":"2021-09-20T19:20:39.397Z","postingPublishTime":"2021-08-21T19:20:39.469Z","postingUpdateTime":"2021-08-21T19:20:39.397Z","requisitionId":"jobWithRequiredFields:474884892813030639","title":"Software engineer","visibility":"ACCOUNT_ONLY"}
    {"completionResults":[{"suggestion":"Google Search","type":"COMPANY_NAME"}],"metadata":{"requestId":"f44d27f8-e5b1-4203-bdc1-4d887f9a1c3a:APAb7ITdtrJFMsXYBlsJwa6lg7r9k5vFrA=="}}
    {"completionResults":[{"suggestion":"Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Developer","type":"JOB_TITLE"},{"suggestion":"Senior Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Engineer","type":"JOB_TITLE"},{"suggestion":"Senior Software Developer","type":"JOB_TITLE"},{"suggestion":"Principal Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Architect","type":"JOB_TITLE"},{"suggestion":"Sr. Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Manager","type":"JOB_TITLE"},{"suggestion":"Software Test Engineer","type":"JOB_TITLE"}],"metadata":{"requestId":"bd088ab1-a7cc-430c-b703-175a43dd5675:APAb7ITJpahfkH3Pa75GukRMcYIUc6Jm+g=="}}
    {"completionResults":[{"suggestion":"Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Developer","type":"JOB_TITLE"},{"suggestion":"Senior Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Engineer","type":"JOB_TITLE"},{"suggestion":"Senior Software Developer","type":"JOB_TITLE"},{"suggestion":"Principal Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Architect","type":"JOB_TITLE"},{"suggestion":"Sr. Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Manager","type":"JOB_TITLE"},{"suggestion":"Software Test Engineer","type":"JOB_TITLE"}],"metadata":{"requestId":"7dc523bc-dbb4-4ed6-ad01-6d68a3948218:APAb7ISeiIy7aiaxKC9gOeUea6NqPDWUbw=="}}
    Job deleted
    Company deleted
    
	at SampleTests.autoCompleteSampleTest(SampleTests.java:55)
</pre></details>

# Java 11
```
#
# jobs/v3
#

[ERROR] Tests run: 12, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 137.238 s <<< FAILURE! - in SampleTests
[ERROR] SampleTests.autoCompleteSampleTest  Time elapsed: 12.747 s  <<< FAILURE!
expected to contain a match for:
    .*completionResults.*"suggestion":"Google","type":"COMPANY_NAME"}.*
    .*completionResults.*"suggestion":"Software Engineer","type":"JOB_TITLE".*
    .*completionResults.*"suggestion":"Software Engineer","type":"JOB_TITLE".*

but was:
    Company generated: GenericData{classInfo=[careerSiteUri, derivedInfo, displayName, eeoText, externalId, headquartersAddress, hiringAgency, imageUri, keywordSearchableJobCustomAttributes, name, size, suspended, websiteUri], {displayName=Google, externalId=company:-1152442932492732939, headquartersAddress=1600 Amphitheatre Parkway Mountain View, CA 94043}}
    Company created: {"derivedInfo":{"headquartersLocation":{"latLng":{"latitude":37.4219891,"longitude":-122.0840944},"locationType":"STREET_ADDRESS","postalAddress":{"addressLines":["1600 Amphitheatre Pkwy, Mountain View, CA 94043, USA"],"administrativeArea":"CA","locality":"Mountain View","postalCode":"94043","regionCode":"US"},"radiusInMiles":7.391165065886471E-5}},"displayName":"Google","externalId":"company:-1152442932492732939","headquartersAddress":"1600 Amphitheatre Parkway Mountain View, CA 94043","name":"projects/java-docs-samples-testing/companies/d6ce55c6-4856-4fca-86f7-f0ece0805f8c"}
    Job generated: GenericData{classInfo=[addresses, applicationInfo, companyDisplayName, companyName, compensationInfo, customAttributes, degreeTypes, department, derivedInfo, description, employmentTypes, incentives, jobBenefits, jobEndTime, jobLevel, jobStartTime, languageCode, name, postingCreateTime, postingExpireTime, postingPublishTime, postingRegion, postingUpdateTime, processingOptions, promotionValue, qualifications, requisitionId, responsibilities, title, visibility], {applicationInfo=GenericData{classInfo=[emails, instruction, uris], {uris=[http://careers.google.com]}}, companyName=projects/java-docs-samples-testing/companies/d6ce55c6-4856-4fca-86f7-f0ece0805f8c, description=Design, develop, test, deploy, maintain and improve software., requisitionId=jobWithRequiredFields:-5869308612335516450, title=Software Engineer}}
    Job created: {"applicationInfo":{"uris":["http://careers.google.com"]},"companyDisplayName":"Google","companyName":"projects/java-docs-samples-testing/companies/d6ce55c6-4856-4fca-86f7-f0ece0805f8c","description":"Design, develop, test, deploy, maintain and improve software.","languageCode":"en","name":"projects/java-docs-samples-testing/jobs/142614115550929606","postingCreateTime":"2021-08-22T01:39:35.871Z","postingExpireTime":"2021-09-21T01:39:35.871Z","postingPublishTime":"2021-08-22T01:39:35.939Z","postingUpdateTime":"2021-08-22T01:39:35.871Z","requisitionId":"jobWithRequiredFields:-5869308612335516450","title":"Software engineer","visibility":"ACCOUNT_ONLY"}
    {"completionResults":[{"suggestion":"Google Search","type":"COMPANY_NAME"}],"metadata":{"requestId":"5f184242-7955-4196-8193-41e3dca697b9:APAb7IRX4uvo1aAH+tBF4KN1ASf3bpW4zg=="}}
    {"completionResults":[{"suggestion":"Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Developer","type":"JOB_TITLE"},{"suggestion":"Senior Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Engineer","type":"JOB_TITLE"},{"suggestion":"Senior Software Developer","type":"JOB_TITLE"},{"suggestion":"Principal Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Architect","type":"JOB_TITLE"},{"suggestion":"Sr. Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Manager","type":"JOB_TITLE"},{"suggestion":"Software Test Engineer","type":"JOB_TITLE"}],"metadata":{"requestId":"dec1f14d-b72b-4062-bb0a-09ec7dc10ae7:APAb7ITCsr4XMfiLtpuTLtzvT5EXqQrxkA=="}}
    {"completionResults":[{"suggestion":"Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Developer","type":"JOB_TITLE"},{"suggestion":"Senior Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Engineer","type":"JOB_TITLE"},{"suggestion":"Senior Software Developer","type":"JOB_TITLE"},{"suggestion":"Principal Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Architect","type":"JOB_TITLE"},{"suggestion":"Sr. Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Manager","type":"JOB_TITLE"},{"suggestion":"Software Test Engineer","type":"JOB_TITLE"}],"metadata":{"requestId":"fa2583b3-bb7c-455b-8c3a-a055a752a20f:APAb7IQRljKlHyfDnXaQAhi6qmrHQreORw=="}}
    Job deleted
    Company deleted

	at SampleTests.autoCompleteSampleTest(SampleTests.java:55)

[ERROR] Failures:
[ERROR]   SampleTests.autoCompleteSampleTest:55 expected to contain a match for:
    .*completionResults.*"suggestion":"Google","type":"COMPANY_NAME"}.*
    .*completionResults.*"suggestion":"Software Engineer","type":"JOB_TITLE".*
    .*completionResults.*"suggestion":"Software Engineer","type":"JOB_TITLE".*

but was:
    Company generated: GenericData{classInfo=[careerSiteUri, derivedInfo, displayName, eeoText, externalId, headquartersAddress, hiringAgency, imageUri, keywordSearchableJobCustomAttributes, name, size, suspended, websiteUri], {displayName=Google, externalId=company:-1152442932492732939, headquartersAddress=1600 Amphitheatre Parkway Mountain View, CA 94043}}
    Company created: {"derivedInfo":{"headquartersLocation":{"latLng":{"latitude":37.4219891,"longitude":-122.0840944},"locationType":"STREET_ADDRESS","postalAddress":{"addressLines":["1600 Amphitheatre Pkwy, Mountain View, CA 94043, USA"],"administrativeArea":"CA","locality":"Mountain View","postalCode":"94043","regionCode":"US"},"radiusInMiles":7.391165065886471E-5}},"displayName":"Google","externalId":"company:-1152442932492732939","headquartersAddress":"1600 Amphitheatre Parkway Mountain View, CA 94043","name":"projects/java-docs-samples-testing/companies/d6ce55c6-4856-4fca-86f7-f0ece0805f8c"}
    Job generated: GenericData{classInfo=[addresses, applicationInfo, companyDisplayName, companyName, compensationInfo, customAttributes, degreeTypes, department, derivedInfo, description, employmentTypes, incentives, jobBenefits, jobEndTime, jobLevel, jobStartTime, languageCode, name, postingCreateTime, postingExpireTime, postingPublishTime, postingRegion, postingUpdateTime, processingOptions, promotionValue, qualifications, requisitionId, responsibilities, title, visibility], {applicationInfo=GenericData{classInfo=[emails, instruction, uris], {uris=[http://careers.google.com]}}, companyName=projects/java-docs-samples-testing/companies/d6ce55c6-4856-4fca-86f7-f0ece0805f8c, description=Design, develop, test, deploy, maintain and improve software., requisitionId=jobWithRequiredFields:-5869308612335516450, title=Software Engineer}}
    Job created: {"applicationInfo":{"uris":["http://careers.google.com"]},"companyDisplayName":"Google","companyName":"projects/java-docs-samples-testing/companies/d6ce55c6-4856-4fca-86f7-f0ece0805f8c","description":"Design, develop, test, deploy, maintain and improve software.","languageCode":"en","name":"projects/java-docs-samples-testing/jobs/142614115550929606","postingCreateTime":"2021-08-22T01:39:35.871Z","postingExpireTime":"2021-09-21T01:39:35.871Z","postingPublishTime":"2021-08-22T01:39:35.939Z","postingUpdateTime":"2021-08-22T01:39:35.871Z","requisitionId":"jobWithRequiredFields:-5869308612335516450","title":"Software engineer","visibility":"ACCOUNT_ONLY"}
    {"completionResults":[{"suggestion":"Google Search","type":"COMPANY_NAME"}],"metadata":{"requestId":"5f184242-7955-4196-8193-41e3dca697b9:APAb7IRX4uvo1aAH+tBF4KN1ASf3bpW4zg=="}}
    {"completionResults":[{"suggestion":"Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Developer","type":"JOB_TITLE"},{"suggestion":"Senior Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Engineer","type":"JOB_TITLE"},{"suggestion":"Senior Software Developer","type":"JOB_TITLE"},{"suggestion":"Principal Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Architect","type":"JOB_TITLE"},{"suggestion":"Sr. Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Manager","type":"JOB_TITLE"},{"suggestion":"Software Test Engineer","type":"JOB_TITLE"}],"metadata":{"requestId":"dec1f14d-b72b-4062-bb0a-09ec7dc10ae7:APAb7ITCsr4XMfiLtpuTLtzvT5EXqQrxkA=="}}
    {"completionResults":[{"suggestion":"Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Developer","type":"JOB_TITLE"},{"suggestion":"Senior Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Engineer","type":"JOB_TITLE"},{"suggestion":"Senior Software Developer","type":"JOB_TITLE"},{"suggestion":"Principal Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Architect","type":"JOB_TITLE"},{"suggestion":"Sr. Software Engineer","type":"JOB_TITLE"},{"suggestion":"Software Development Manager","type":"JOB_TITLE"},{"suggestion":"Software Test Engineer","type":"JOB_TITLE"}],"metadata":{"requestId":"fa2583b3-bb7c-455b-8c3a-a055a752a20f:APAb7IQRljKlHyfDnXaQAhi6qmrHQreORw=="}}
    Job deleted
    Company deleted

[ERROR] Tests run: 12, Failures: 1, Errors: 0, Skipped: 0
```

--- a/jobs/v3/src/test/java/SampleTests.java
+++ b/jobs/v3/src/test/java/SampleTests.java
@@ -54,7 +54,7 @@ public class SampleTests {
     assertThat(bout.toString())
         .containsMatch(
             ".*completionResults.*\"suggestion\":"
-                + "\"Google\",\"type\":\"COMPANY_NAME\"}.*\n"
+                + "\"Google Search\",\"type\":\"COMPANY_NAME\"}.*\n"
                 + ".*completionResults.*\"suggestion\""
                 + ":\"Software Engineer\",\"type\":\"JOB_TITLE\".*\n"
                 + ".*completionResults.*\"suggestion\""

